docs/data-sources/equinix_fabric_connection.md

```diff
@@ -0,0 +1,418 @@
+- `peering_type` (String)
+- `port` (Set of Object) (see [below for nested schema](#nestedobjatt--z_side--access_point--port))
+- `profile` (Set of Object) (see [below for nested schema](#nestedobjatt--z_side--access_point--profile))
+- `provider_connection_id` (String)
+- `seller_region` (String)
+- `type` (String)
+- `virtual_device` (Set of Object) (see [below for nested schema](#nestedobjatt--z_side--access_point--virtual_device))
+
+<a id="nestedobjatt--z_side--access_point--account"></a>
+### Nested Schema for `z_side.access_point.account`
+
+Read-Only:
+
+- `account_name` (String)
+- `account_number` (Number)
+- `global_cust_id` (String)
+- `global_org_id` (String)
+- `global_organization_name` (String)
+- `org_id` (Number)
+- `organization_name` (String)
+
+
+<a id="nestedobjatt--z_side--access_point--gateway"></a>
+### Nested Schema for `z_side.access_point.gateway`
+
+Read-Only:
+
+- `href` (String)
+- `state` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--z_side--access_point--interface"></a>
+### Nested Schema for `z_side.access_point.interface`
+
+Read-Only:
+
+- `id` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--z_side--access_point--link_protocol"></a>
+### Nested Schema for `z_side.access_point.link_protocol`
+
+Read-Only:
+
+- `type` (String)
+- `vlan_c_tag` (Number)
+- `vlan_s_tag` (Number)
+- `vlan_tag` (Number)
+
+<a id="nestedobjatt--z_side--access_point--location"></a>
+### Nested Schema for `z_side.access_point.location`
+
+Read-Only:
+
+- `ibx` (String)
+- `metro_code` (String)
+- `metro_name` (String)
+- `region` (String)
+
+
+<a id="nestedobjatt--z_side--access_point--port"></a>
+### Nested Schema for `z_side.access_point.port`
+
+Read-Only:
+
+- `href` (String)
+- `name` (String)
+- `redundancy` (Set of Object) (see [below for nested schema](#nestedobjatt--z_side--access_point--port--redundancy))
+- `uuid` (String)
+
+<a id="nestedobjatt--z_side--access_point--port--redundancy"></a>
+### Nested Schema for `z_side.access_point.port.uuid`
+
+Read-Only:
+
+- `priority` (String)
+
+
+
+<a id="nestedobjatt--z_side--access_point--profile"></a>
+### Nested Schema for `z_side.access_point.profile`
+
+Read-Only:
+
+- `description` (String)
+- `href` (String)
+- `name` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--z_side--access_point--virtual_device"></a>
+### Nested Schema for `z_side.access_point.virtual_device`
+
+Read-Only:
+
+- `href` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+
+<a id="nestedatt--z_side--service_token"></a>
+### Nested Schema for `z_side.service_token`
+
+Read-Only:
+
+- `description` (String)
+- `href` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+
+<a id="nestedatt--account"></a>
+### Nested Schema for `account`
+
+Read-Only:
+
+- `account_name` (String)
+- `account_number` (Number)
+- `global_cust_id` (String)
+- `global_org_id` (String)
+- `global_organization_name` (String)
+- `org_id` (Number)
+- `organization_name` (String)
```

Comment:
More arguments without description, but again I see all them in the schemas definition

Reply:
Generally if the names are self explanatory, I think detailed description is ignored in the spec, but we need to inform team.